test/functional/tests/lazy_writes/recovery/test_recovery_unplug.py

```diff
@@ -148,7 +151,12 @@ def test_recovery_unplug_cache_raw(cache_mode, cls):
                             f"{cache.get_dirty_blocks().get_value(Unit.Blocks4096)}")
 
     with TestRun.step("Stop cache."):
-        cache.stop()
+        output = TestRun.executor.run(
+            stop_cmd(cache_id=str(cache.cache_id), no_data_flush=True, shortcut=False))
+        if output.exit_code != 0:
+            cli_messages.check_stderr_msg(output, cli_messages.stop_cache_errors)
+        else:
+            TestRun.LOGGER.warning("Expected stopping cache with errors.")
```

Comment:
Try not to use api.cas.cli, if you don't necessarily need it.
`cache.stop(no_data_flush=True)` would suffice, and it throws a nice CmdException that contains output if exit_code is not 0.